### PR TITLE
allow use of ITERATIONS argument in make test-cmd

### PR DESCRIPTION
**Pull request by runyontr**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Allow for use of ITERATION argument in `make test-cmd WHAT="authentication" ITERATIONS=5`

**Which issue(s) this PR fixes**:

None

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

**Comment by spiffxp** [comment]
Can you rebase/squash/etc. to get the merge commits out of here?

/uncc @gmarek @vishh 
/cc @neolit123 
for the kubeadm change, send over to someone more appropriate if your plate is too full

**Comment by spiffxp** [comment]
/unassign @zmerlynn 
/assign @spiffxp @BenTheElder 
assigning ben as well to cover the build/ changes

File: test/cmd/kubeadm.sh [hunk 1]
```diff
@@ -32,8 +32,8 @@ run_kubeadm_tests() {
   # kubernetes/kubeadm/issues/1430 is fixed
   make -C "${KUBE_ROOT}" test \
   WHAT=k8s.io/kubernetes/cmd/kubeadm/test/cmd \
-  KUBE_TEST_ARGS="--kubeadm-path '${KUBEADM_PATH}'" \
-  KUBE_TIMEOUT="--timeout=600s"
+  KUBE_TEST_ARGS="--kubeadm-path '${KUBEADM_PATH}' --count '${ITERATIONS:-1}'" \
+  KUBE_TIMEOUT="--timeout 600s"
```
Comment (BenTheElder) [inline test/cmd/kubeadm.sh]: why this change?

**Review by neolit123 — COMMENTED** [review]
LGTM on the kubeadm related change.
thanks.
